<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarRent</class>
 <widget class="QWidget" name="CarRent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>910</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CarRent Admin Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>CarRent Admin 1.0</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_8">
              <property name="title">
               <string>Users</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QTableView" name="tblUsers">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_10">
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Edit - Create</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <item>
                   <widget class="QGroupBox" name="groupBox_4">
                    <property name="title">
                     <string>ID</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_5">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedUserId">
                       <property name="enabled">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_12">
                    <property name="title">
                     <string>Name</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedUserName"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_13">
                    <property name="title">
                     <string>Surname</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedUserSurname"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_14">
                    <property name="title">
                     <string>Address</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_8">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedUserAddress"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_15">
                    <property name="title">
                     <string>Credit Card</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_9">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedUserCredtCard"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_16">
                    <property name="title">
                     <string>Driving Licence</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_10">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedUserDrivingLic"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_3">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>45</height>
                     </size>
                    </property>
                    <property name="title">
                     <string/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="btnUserCreate">
                       <property name="maximumSize">
                        <size>
                         <width>100</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>CREATE</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="btnUserDelete">
                       <property name="maximumSize">
                        <size>
                         <width>100</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>DELETE</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="btnEditUser">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>EDIT</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_9">
              <property name="title">
               <string>Cars</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QTableView" name="tblCars"/>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_11">
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Edit - Create</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_18">
                  <item>
                   <widget class="QGroupBox" name="groupBox_22">
                    <property name="title">
                     <string>ID</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_16">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedCarId">
                       <property name="enabled">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_21">
                    <property name="title">
                     <string>Name</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_15">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedCarName"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_18">
                    <property name="title">
                     <string>Brand</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_13">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedCarBrand"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_23">
                    <property name="title">
                     <string>Tail Number</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_17">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedCarTail"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_17">
                    <property name="title">
                     <string>Class</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_12">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="nedCarClass"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_19">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>45</height>
                     </size>
                    </property>
                    <property name="title">
                     <string/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="btnCarCreate">
                       <property name="maximumSize">
                        <size>
                         <width>100</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>CREATE</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="btnCarDelete">
                       <property name="maximumSize">
                        <size>
                         <width>100</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>DELETE</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="btnCarUser">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>EDIT</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="maximumSize">
            <size>
             <width>450</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Rents</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTableWidget" name="tableRents"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnRefresh">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEXIT">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>EXIT</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
